selenium_tests/cross_listing/page_objects/cross_listing_base_page_object.py

```diff
@@ -0,0 +1,34 @@
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+
+from selenium_common.base_page_object import BasePageObject
+
+
+class Locators(object):
+    @classmethod
+    def TD_TEXT_XPATH(cls, search_text):
```

Comment:
We should probably move this (eventually?) into the base page object, since it is used in so many different tests and is general enough to match a `<td>` on any page.